2048/2048/src/main/java/com/appcoins/eskills2048/PlayerRankingAdapter.java

```diff
@@ -0,0 +1,69 @@
+package com.appcoins.eskills2048;
+
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.appcoins.eskills2048.model.User;
+
+import java.util.List;
+
+public class PlayerRankingAdapter extends RecyclerView.Adapter<PlayerRankingAdapter.ViewHolder> {
+  private List<User> localDataSet;
+  private final Context context;
```

Comment:
no need to keep a reference to context

Reply:
i'm using it for getting a resources string inside `onBindViewHolder`. what should i do instead?

Reply:
if you accept my changes, you'll no longer need it, but you can get the resources from the itemview(from viewholder) since it has the context already